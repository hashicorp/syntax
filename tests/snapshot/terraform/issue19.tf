variable "test" {
  type = object({
    test_object = map( # here
      object({ # here
        test_map = string
      })
    )
  })
}