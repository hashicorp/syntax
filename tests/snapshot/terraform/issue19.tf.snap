>variable "test" {
#^^^^^^^^ source.hcl.terraform meta.block.hcl entity.name.type.terraform
#        ^ source.hcl.terraform meta.block.hcl
#         ^^^^^^ source.hcl.terraform meta.block.hcl variable.other.enummember.hcl
#               ^ source.hcl.terraform meta.block.hcl
#                ^ source.hcl.terraform meta.block.hcl punctuation.section.block.begin.hcl
>  type = object({
#^^ source.hcl.terraform meta.block.hcl
#  ^^^^ source.hcl.terraform meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#      ^ source.hcl.terraform meta.block.hcl variable.declaration.hcl
#       ^ source.hcl.terraform meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#        ^ source.hcl.terraform meta.block.hcl variable.declaration.hcl
#         ^^^^^^ source.hcl.terraform meta.block.hcl storage.type.hcl
#               ^ source.hcl.terraform meta.block.hcl punctuation.section.parens.begin.hcl
#                ^ source.hcl.terraform meta.block.hcl meta.braces.hcl punctuation.section.braces.begin.hcl
>    test_object = map( # here
#^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#    ^^^^^^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#               ^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#                ^ source.hcl.terraform meta.block.hcl meta.braces.hcl keyword.operator.assignment.hcl
#                 ^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#                  ^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl storage.type.hcl
#                     ^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#                       ^ source.hcl.terraform meta.block.hcl meta.braces.hcl comment.line.number-sign.hcl punctuation.definition.comment.hcl
#                        ^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl comment.line.number-sign.hcl
>      object({ # here
#^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#      ^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl storage.type.hcl
#            ^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#             ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl punctuation.section.braces.begin.hcl
#              ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl
#               ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl comment.line.number-sign.hcl punctuation.definition.comment.hcl
#                ^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl comment.line.number-sign.hcl
>        test_map = string
#^^^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl
#        ^^^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#                ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl
#                 ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl keyword.operator.assignment.hcl
#                  ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl
#                   ^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl storage.type.hcl
>      })
#^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl
#      ^ source.hcl.terraform meta.block.hcl meta.braces.hcl meta.braces.hcl punctuation.section.braces.end.hcl
#       ^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
>    )
#^^^^^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
>  })
#^^ source.hcl.terraform meta.block.hcl meta.braces.hcl
#  ^ source.hcl.terraform meta.block.hcl meta.braces.hcl punctuation.section.braces.end.hcl
#   ^ source.hcl.terraform meta.block.hcl punctuation.section.parens.end.hcl
>}
#^ source.hcl.terraform meta.block.hcl punctuation.section.block.end.hcl